modules/ibmz-configure-devices-mco.adoc

```diff
@@ -0,0 +1,263 @@
+// Module included in the following assemblies:
+//
+// * post-installation-configuration/ibmz-post-install.adoc
+
+:_content-type: PROCEDURE
+[id="configure-additional-devices-using-mco_{context}"]
+= Configuring additional devices using the Machine Config Operator (MCO) 
+
+Tasks in this section describe how to use features of the Machine Config Operator to configure additional devices in an {ibmzProductName} or LinuxONE environment. Configuring devices with the MCO is persistent but only allows specific configurations for compute nodes. MCO does not allow control plane nodes to have different configurations.
```

Comment:
Since this is a CP I'll note this here: we should probably call out the MCO abbreviation after the first mentioning. That way, we ensure users know what it is, and the abbreviation is close by (so that they do not have to search on the page for it).